### ISSUE TITLE
Failing UI tests

### ISSUE THREAD

**Issue by Baron-Severin** [mozilla-mobile/firefox-tv#1083]
UI Tests are currently broken.

Four compilation failures
- [ ] PinnedTileTests#testCustomPinnedTile
- [ ] TvScreenshots#defaultHomeScreen
- [ ] TvScreenshots#browserNavOverlay
- [ ] BrowserOverlayTest#testBrowserOverlay
- [ ] PageLoadTest#PageLoadTest

When these compilation failures are commented out, 11 of our 12 tests fail.

We need to go through all tests, determine whether or not each is still a valid test case, then update them.  For this story to be considered complete, all tests should compile and pass.

**Comment by AaronMT** [comment]
In-progress.

**Comment by liuche** [comment]
Manual testing is still going on, so let's move this into 3.1 - it's a nice-to-have but seems to be blocked on some other things that aren't in the 3.0 milestone.
